dhis2-android-app/src/main/java/org/dhis2/mobile/utils/TextFileUtils.java

```diff
@@ -115,6 +127,32 @@ public static void writeTextFile(Context context, Directory dir, FileNames name,
         writeTextFile(context, dir, name.toString(), data);
     }
 
+
+
+    public static void writeLineIntoTextFile(Context context, Directory dir,
```

Comment:
this code has duplicate code with writeTextFile, can you refactor writeTextFile according to parameter for add formatted formattedDate after text content to add and remove this method?